robyn/scaffold/structured/no-db/api/handlers/sample.py

```diff
@@ -0,0 +1,13 @@
+from robyn import SubRouter
+
+router = SubRouter(__name__, "/sample")
+
+
+class SampleHandlers:
+    @router.post("/one")
+    @staticmethod
+    def one(): ...
+
+    @router.get("/two")
+    @staticmethod
+    def two(): ...
```

Comment:
Why are we confining the two methods in a namespace?

Reply:
Having the two methods in a class is just a user preference, and doesn't have any impact on the actual routing.  Updated docs to explicitly mention this, also, added an example handler outside the class


```python

class SampleHandlers:
    """
    note: the handles being grouped in a class like this is complete optional, and doesn't have any impact on routing
    """
    @router.post("/one")
    @staticmethod
    def one(): ...

    @router.get("/two")
    @staticmethod
    def two(): ...


@router.get("three/")
def three():
    return {}
```

Reply:
@ashupednekar , let's not do this. I am not a big fan of this syntax